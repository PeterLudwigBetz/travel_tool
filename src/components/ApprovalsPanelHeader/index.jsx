import React, { PureComponent } from 'react';
import PageHeader from '../PageHeader';
import ButtonsGroup from '../button-group/ButtonGroup';
import HeaderPagination from '../Pagination/HeaderPagination';
import '../RequestPanelHeader/Request.scss';

class ApprovalsPanelHeader extends PureComponent {
  render() {

    return (
      <div className="request-panel-header">
        <PageHeader title="APPROVALS" />
        <div className="open-requests">
          <ButtonsGroup buttonsType="approvals" />
          <HeaderPagination />
        </div>
      </div>
    );
  }
}

export default ApprovalsPanelHeader;
