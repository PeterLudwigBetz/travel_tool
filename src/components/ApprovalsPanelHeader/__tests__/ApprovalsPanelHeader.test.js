import React from 'react';
import ApprovalsPanelHeader from '../index';

it('should render approvals header correctly', () => {
    const wrapper =  shallow(<ApprovalsPanelHeader />);
    expect(wrapper).toMatchSnapshot();
});

