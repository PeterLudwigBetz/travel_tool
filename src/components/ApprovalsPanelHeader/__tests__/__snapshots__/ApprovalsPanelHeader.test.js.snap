// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render approvals header correctly 1`] = `
<div
  className="request-panel-header"
>
  <PageHeader
    actionBtn=""
    title="APPROVALS"
    toggleNewRequestModal={[Function]}
  />
  <div
    className="open-requests"
  >
    <ButtonGroup
      buttonsType="approvals"
      fetchUserRequests={null}
      limit={10}
      openRequestsCount={null}
      pastRequestsCount={null}
    />
    <HeaderPagination
      getRequestsWithLimit={null}
    />
  </div>
</div>
`;
